packages/suite/src/components/wallet/TransactionItem/TransactionItem.tsx

```diff
@@ -254,7 +243,7 @@ export const TransactionItem = memo(
                     </TxTypeIconWrapper>
 
                     <Content>
```

Comment:
It would be nice to separate some of the components in this return to make it more readable. But I understand it was here before, so it's up to you.

Reply:
I will leave it as is for now, I need this PR to be merged to work on NFT section that is higher priority